frontendActionIds/src/main/java/com/virtuslab/gitmachete/frontend/actionids/ActionPlaces.java

```diff
@@ -4,5 +4,6 @@
   private ActionPlaces() {}
 
   public static final String ACTION_PLACE_CONTEXT_MENU = "GitMacheteContextMenu";
+  public static final String ACTION_PLACE_EMPTY_TEXT = "GitMacheteEmptyText";
```

Comment:
Is this an "empty text", or rather a "non-empty text displayed in case of empty machete file"? :thinking:

Reply:
The naming is unusual but it comes from `com.intellij.ui.table.JBTable#myEmptyText`. Hence "empty" in "empty text" not about "text" but about the table. And at this level (JBTable) a name like "non-empty text displayed in case of empty table" seems to be more descriptive.

Reply:
But despite their naming issues I will rename `emptyText`-related things.